code/game/objects/items/twohanded.dm

```diff
@@ -778,11 +778,11 @@
 	throwforce = 20
 	throw_speed = 4
 	attack_verb = list("gored")
-	var/clonechance = 50
+	var/clonechance = 75
```

Comment:
god fuck no please 

clearly you haven't seen the entire station be killed by these before.